test/unit/provision/synchronousButtonProvision.json

```diff
@@ -28,6 +28,18 @@
           "name": "op_result",
```

Comment:
Same as above. The IOTA doesn't make any checks about what attributes are provisioned, that's dependant on the scenario. The important thing here is to have the initial entity created with the active attributes and the static attributes added to any updateContext operation to the CB. The particular attributes used are scenario-dependant.